snippets/createClass_react_native.sublime-snippet

```diff
@@ -2,12 +2,14 @@
   <content><![CDATA[
 'use strict';
 
-import React, {
+import React, { Component } from 'react';
```

Comment:
@shiv86 can you change this line to just `import { Component } from 'react';`? Code linters will complain about the unused variable because it isn't used in the snippet.